addons/web/static/src/webclient/webclient.js

```diff
@@ -99,7 +99,7 @@ export class WebClient extends Component {
         // we let the browser do the default behavior and
         // we do not want any other listener to execute.
         if (
-            ev.ctrlKey &&
+            ev.ctrlKey && !ev.target.contentEditable &&
```

Comment:
I'm surprised the linter let you do that. I thought the only valid forms where:
```js
if (every && condition && in && one && line) {
    ...
} else if (
    every &&
    condition &&
    on &&
    separate &&
    lines
) {
    ...
}
```

Reply:
Linter be like: 
"I'll allow it" :judge: 

Changed